startup/serializers.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework import serializers
+from .models import startup
+
+class startupSerializer(serializers.ModelSerializer):
```

Comment:
`StartupSerializer` @naveennvrgup

Reply:
these are the commits of @anshsrtv. i hav just cleaned his old PR #33

Reply:
sir this is cleanup PR of PR #33 made by @anshsrtv. for some reason he have deleted these files on commit d8f9e95(if u check PR #33) for which u have been asking changes for. So these files dont exit in this branch. I just reset --soft to the commit he added the outdated frontend files and removed them and made a PR.